libpod/pod.go

```diff
@@ -292,3 +294,36 @@ func (p *Pod) GetPodStats(previousContainerStats map[string]*ContainerStats) (ma
 	}
 	return newContainerStats, nil
 }
+
+// CreatePortBindings iterates ports mappings and exposed ports into a format CNI understands
+func CreatePortBindings(ports []string) ([]ocicni.PortMapping, error) {
```

Comment:
Why'd this move? Can it go int cmd/podman/shared?

Reply:
it moved for varlink but yes, shared is the right place for it